src/normalize-keyframes.js

```diff
@@ -150,6 +150,10 @@
     return value;
   }
 
+  function isPermittedKeyframeProperty(property) {
+    return property !== 'display' && property.lastIndexOf('animation', 0) === -1;
```

Comment:
lastIndexOf(, 0) searches backwards from 0, meaning only one offset is tested. If the property name is long, indexOf needs to check many offsets.